jax/_src/numpy/reductions.py

```diff
@@ -443,11 +443,12 @@ def _var(a: ArrayLike, axis: Axis = None, dtype: DTypeLike = None,
 
   if where is None:
     if axis is None:
-      normalizer = core.dimension_as_value(np.size(a))
+      normalizer = float(core.dimension_as_value(np.size(a)))
```

Comment:
It looks like this is causing failures in the dynamic shapes tests because `float` requires a static input. A better option may be to cast this (dynamically) to the type of the output.

Reply:
Resolved